app/src/main/java/com/nutomic/syncthingandroid/service/RestApi.java

```diff
@@ -311,7 +330,13 @@ private void removeFolderInternal(String id) {
     }
 
     public Device getLocalDevice() {
-        for (Device d : getDevices(true)) {
+        List<Device> devices = getDevices(true);
+        if (devices.isEmpty()) {
+            Log.e(TAG, "getLocalDevice: devices is empty");
+            throw new RuntimeException();
```

Comment:
That's not a very useful exception...

Reply:
That's still not a useful exception, regardless that the error message is good.
Error messages will not be sent to google play.

Reply:
Ok got the point. So we need a good error message thrown "to google play". I'm going to improve this shorty ...

Reply:
I put the log message to be thrown as RuntimeException. commit https://github.com/syncthing/syncthing-android/pull/1119/commits/9bdc1a7ebf72bd563bf2e7b2340479db9ead5d97